command/server.go

```diff
@@ -1298,6 +1298,21 @@ func (c *ServerCommand) Run(args []string) int {
 	info := make(map[string]string)
 	info["log level"] = logLevelString
 	infoKeys = append(infoKeys, "log level")
+
+	// returns a slice of env vars formatted as "key=value"
+	envVars := os.Environ()
+	var envVarKeys []string
+	for _, v := range envVars {
+		splitEnvVars := strings.Split(v, "=")
+		envVarKeys = append(envVarKeys, splitEnvVars[0])
```

Comment:
This formatting looks wrong, can you run `make fmt` in the root directory and commit the changes (if any)?

Reply:
that's weird. It looks way more indented in GH than on my local.